search-commons/src/main/java/no/unit/nva/search/resource/Constants.java

```diff
@@ -122,7 +122,8 @@ public final class Constants {
     public static final String COURSE_CODE_KEYWORD =
         ENTITY_PUBLICATION_CONTEXT_DOT + COURSE + DOT + CODE + DOT + KEYWORD;
 
-    public static final String ENTITY_DESCRIPTION_PUBLICATION_PAGES = ENTITY_PUBLICATION_INSTANCE_DOT + "pages.pages";
+    public static final String ENTITY_DESCRIPTION_PUBLICATION_PAGES =
+        ENTITY_PUBLICATION_INSTANCE_DOT + "pages.pages.keyword";
```

Comment:
keyword? did it not work before?

Reply:
no, it only behaved as a number when using keyword